packages/flutter_tools/lib/src/commands/create_base.dart

```diff
@@ -215,7 +227,7 @@ abstract class CreateBase extends FlutterCommand {
   /// If `--org` is specified in the command, returns that directly.
   /// If `--org` is not specified, returns the organization from the existing project.
   @protected
-  Future<String> getOrganization(Directory projectDir) async {
+  Future<String> getOrganization() async {
```

Comment:
@jmagman Since this method returns a future and it's not a constant lookup, I decided not making this a getter based on 
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#getters-feel-faster-than-methods